web-ui/view/index.ml

```diff
@@ -13,56 +13,41 @@ let rows prefix orgs =
   let orgs = List.sort compare orgs in
   List.fold_left (fun l org -> List.append l [ row ~org ]) [] orgs
 
-let list_orgs prefix orgs =
-  Template.instance
-    [
-      Tyxml.Html.script ~a:[ a_src "/js/index-page-org-search.js" ] (txt "");
-      div
-        ~a:
+let search_bar_options = function
+  | [] | [ _ ] -> []
+  | orgs ->
+      let build_option =
+        option ~a:[ a_value "all" ] (txt "all")
+        :: List.map
+             (fun (prefix, name, _) -> option ~a:[ a_value prefix ] (txt name))
```

Comment:
This isn't right, it generates an `option` from every single org. If you have multiple GitHub users for example there will be a GitHub option for every single one.

Reply:
Yep you're right, my mistake